cmd/lytfs/main.go

```diff
@@ -28,7 +31,13 @@ func main() {
 	ctx, cancel := context.WithTimeout(context.Background(), *timeout)
 	defer cancel()
 
-	sources := yagfuncdata.DefaultSources(yagfuncdata.DefaultFileContentProvider)
+	var sources []yagfuncdata.Source
+	if token := os.Getenv("YAGFUNCDATA_AUTH_TOKEN"); token != "" {
+		fcp := yagfuncdata.NewGitHubFileProvider(github.NewClient(nil).WithAuthToken(token), "botlabs-gg", "yagpdb", "master")
+		sources = yagfuncdata.DefaultSources(fcp)
+	} else {
+		sources = yagfuncdata.DefaultSources(yagfuncdata.DefaultFileContentProvider)
+	}
```

Comment:
This is a nit, but I would rather write
```suggestion
	fcp := yagpdbfundata.DefaultFileContentProvider
	if token := os.Getenv("YAGFUNCDATA_AUTH_TOKEN"); token != "" {
		fcp = yagfuncdata.NewGitHubFileProvider(github.NewClient(nil).WithAuthToken(token), "botlabs-gg", "yagpdb", "master")
	}
	sources := yagfuncdata.DefaultSources(fcp)
```

Reply:
Sure, that reads cleaner anyway.